web-console/src/views/ingestion-view/ingestion-view.tsx

```diff
@@ -1138,6 +1163,17 @@ ORDER BY "rank" DESC, "created_time" DESC`;
                 localStorageKey={LocalStorageKeys.TASKS_REFRESH_RATE}
                 onRefresh={auto => this.taskQueryManager.rerunLastQuery(auto)}
               />
+              {/* Toggle switch to query overload or RDS(RDS network throughput increases 3X on firing the UI) */}
```

Comment:
RDS is an implementation detail, and the volumes depend on the cluster, so let's make this comment more generic in case we decide to include this patch upstream.
```suggestion
              {/* Toggle switch to prefer overload over SQL since SQL can affect metadata store performance with large number of tasks) */}
```

Reply:
ideally we should also file an issue upstream related to the metadata store query performance and link to the issue in the comment